tests/bugfixes/github/test_issue_283.py

```diff
@@ -0,0 +1,183 @@
+# -*- coding: utf-8 -*-
+
+import system_tests
+import os.path
+
+class TestFirstPoC(metaclass=system_tests.CaseMeta):
+    """
+    Regression test for the bug described in:
+    https://github.com/Exiv2/exiv2/issues/283
+    """
+    url = "https://github.com/Exiv2/exiv2/issues/283"
+
+    filename = os.path.join("$data_path", "pocIssue283.jpg")
+    commands = ["$exiv2 " + filename]
```

Comment:
You should be able to write `"$exiv2 $filename"` now (the manual concatenation is no longer required).